config.yaml

```diff
@@ -457,6 +467,9 @@ orgs:
           aap_configuration_template: admin
           aap_utilities: admin
           ah_configuration: admin
+          cloud.aws_ops: admin
```

Comment:
These repositories are not yet available

Reply:
Correct, they need to be migrated from ansible-collections, my understanding was the process was easier if I had this PR pre-prepared